### PR TITLE
Create Website Footer

### PR DESCRIPTION
**Pull request by iomerkoyuncu**
Hi, I'm İsmet, I want to help with this issue.

I added "flex flex-col justify-between h-screen" to layout div on line 6. To anchor the footer to the bottom of the page.

Navbar
Any Page / Content
Footer

### CODE REVIEW

**Comment by ChristianBingman** [comment]
Please add a comment to this issu rso I can assign you #168

File: src/pages/layout.js [hunk 1]
```diff
@@ -14,8 +14,37 @@ export default function Home(){
...
+                            <li><Link to="/about">Facebook</Link></li>
+                            <li><Link to="/about">Youtube</Link></li>
+                            <li><Link to="/about">Twitter</Link></li>
+                            <li><Link to="/about">LinkedIn</Link></li>
+                        </ul>
+                    </div>
+                </div>
+                <div className='text-center'>
+                    <p className='text-[12px] '>Copyright © Association for Computing Machinery 2023</p>
+                    <p className='text-[12px]'>License is located on this sites Github Repo.</p>
```
Comment (ChristianBingman) [inline src/pages/layout.js:45]: Github Repo should be a link

File: src/pages/layout.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import './layout.css'
 import { Link, Outlet } from 'react-router-dom'
 
-export default function Home(){
+export default function Home() {
     return (
-        <div>
+        <div className='flex flex-col justify-between h-screen'>
```
Comment (ChristianBingman) [inline src/pages/layout.js:6]: This styling shouldn't go over everything

**Review by ChristianBingman — CHANGES_REQUESTED** [review]
Please remove the package-lock.json and just a couple changes for now.

**Comment by iomerkoyuncu** [comment]
Hi again, I added link to github repo. Deleted package.json. Also removed the "h-screen" and other classes.

**Comment by iomerkoyuncu** [comment]
The last commit is for more proper commit message. :)

**Comment by ChristianBingman** [comment]
It looks like you deleted the whole package-lock.json file. You need to re-add the original back to the repo. You can do this with `git checkout main -- package-lock.json`

**Review by ChristianBingman — APPROVED** [review]
Does not properly wrap on mobile, but for time purposes I will go ahead and fix the rest. Thank you!

**Pull request merged**